### PR TITLE
Added support for cancel

### PR DESCRIPTION
**Pull request by trajano**
This is my first cut, I can't get it working with `Task` so I had to resort to `any`.   @sindresorhus  if you can provide some tips to remove it please let me know so I can update the PR.

For  #76

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,15 +11,16 @@
 		"node": ">=12"
 	},
 	"scripts": {
-		"build": "del dist && tsc",
+		"build": "rimraf dist && tsc",
```
Comment (sindresorhus) [inline package.json:14]: Don't do unrelated changes.

File: source/index.ts [hunk 2]
```diff
@@ -281,6 +281,15 @@ export default class PQueue<QueueType extends Queue<RunFunction, EnqueueOptionsT
 		return Promise.all(functions.map(async function_ => this.add(function_, options)));
 	}
 
+	/**
+	Cancels an task from the queue.
+
+	@param fn task to remove from the queue
```
Comment (sindresorhus) [inline source/index.ts:287]: This syntax is not correct.

File: source/priority-queue.ts [hunk 3]
```diff
@@ -43,6 +45,14 @@ export default class PriorityQueue implements Queue<RunFunction, PriorityQueueOp
 		).map((element: Readonly<{run: RunFunction}>) => element.run);
 	}
 
+	remove(fn: () => unknown): void {
+		for (let i = 0; i < this._queue.length; ++i) {
```
Comment (sindresorhus) [inline source/priority-queue.ts:49]: Use a `for-of` loop.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
Sorry, I have a lot of PRs to review. I don't have the time to look into your problem. It's up to you to solve it if you want this to be merged. In general, you need to spend more effort on this PR, both code and docs-wise.

File: package.json (hunk 1 above)
Comment (trajano) [reply package.json:14]: Using `del` causes problems on a Windows build just an FYI.
